integration_tests/base_routes.py

```diff
@@ -527,6 +527,25 @@ async def async_body_post(request: Request):
     return request.body
 
 
+# JSON Request
+
+
+@app.post("/sync/request_json")
+def sync_json_post(request: Request):
+    try:
+        return type(request.json())
```

Comment:
Hey @IdoKendo 👋 

Thank you for the PR 😄 

But can we do `request.body.json()` instead of `request.json()`? In my mind `request.json()` returns a json version of the request object and not request.body

Reply:
I don't think we can do it request.body.json() since body is a string/bytes and not an object we instantiate - Anyway I don't think request.json() implies the request object. That's how it works in both Flask's request object as well.

Reply:
That does make sense. Sounds good. Just one last suggestion for the docstring and then we can merge it :D